superset/connectors/base/models.py

```diff
@@ -185,6 +186,34 @@ def data(self):
             'verbose_map': verbose_map,
         }
 
+    @staticmethod
+    def filter_values_handler(
+            values, target_column_is_numeric=False, is_list_target=False):
+        def handle_single_value(v):
+            # backward compatibility with previous <select> components
+            if isinstance(v, basestring):
+                v = v.strip().strip("'").strip('"')
+                if target_column_is_numeric:
+                    # For backwards compatibility and edge cases
+                    # where a column data type might have changed
+                    v = utils.string_to_num(v)
+            if v == '<NULL>':
+                return None
+            elif v == '<empty string>':
+                return ''
+            return v
+        if isinstance(values, (list, tuple)):
+            values = [handle_single_value(v) for v in values]
+        values = handle_single_value(values)
+        if is_list_target and not isinstance(values, (tuple, list)):
+            values = [values]
```

Comment:
What if `is_list_target` is true but `values` is a tuple? In this case it would return a tuple, is that ok?

Reply:
The variable should probably be called `is_iterable_target`, both tuple and list will work here.